src/database.js

```diff
@@ -0,0 +1,19 @@
+import { MongoClient } from 'mongodb';
+
+let connection = null;
+
+const url = 'mongodb://localhost:27017';
+const dbName =
+  process.env.NODE_ENV !== 'test' ? 'api-particulier' : 'api-particulier-test';
+
+export const getConnection = async () => {
```

Comment:
It seems to me that ` getConnection `isn't explicit enough. Maybe `GetTockensFromDatabase` ?